examples.json

```diff
@@ -208,6 +208,15 @@
     ]
   },
   {
+    "name": "emoji-substitution",
+    "description": "Replaces words with emojis.",
+    "url": "https://github.com/mdn/webextensions-examples/tree/master/emoji-substitution",
+    "manifest_keys": [
+      "content_scripts"
+    ],
+    "javascript_modules": []
+  },
+  {
```

Comment:
I wasn't sure if this file was auto-generated or not, but I added Emoji Substitution here for good measure.